src/generate.js

```diff
@@ -16,12 +16,14 @@ self.generate = function(options) {
 	if (!options.hasOwnProperty('symbols')) options.symbols = false;
 	if (!options.hasOwnProperty('uppercase')) options.uppercase = true;
 	if (!options.hasOwnProperty('excludeSimilarCharacters')) options.excludeSimilarCharacters = false;
+	if (!options.hasOwnProperty('repeatCharactersLimit')) options.repeatCharactersLimit = 0;
 
 	var lowercase = 'abcdefghijklmnopqrstuvwxyz',
 		uppercase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',
 		numbers = '0123456789',
 		symbols = '!@#$%^&*()+_-=}{[]|:;"/?.><,`~',
-		similarCharacters = /[ilLI|`oO0]/g;
+		similarCharacters = /[ilLI|`oO0]/g,
+		repeatingCharacters = options.repeatCharactersLimit ? new RegExp('(.)\\1{' + (options.repeatCharactersLimit) + ',}', 'g') : undefined;
```

Comment:
Do you think it'd be better to put `options.repeatCharactersLimit > 1` here? Or should we throw on `1`?